docs/guide/docs/developers/create-module.md

```diff
@@ -317,7 +317,14 @@ export default class Database {
 
 ### Migration
 
-Database migration isn't available at the moment, it should be added in a future iteration
+The method `bp.database.runMigrations` can run your module migration.
```

Comment:
`bp.database.runMigrations` method can be used to run module-specific migrations.